tasks/main.yml

```diff
@@ -60,7 +67,7 @@
         state: present
     - name: Add Docker apt-repository
       apt_repository:
-        repo: 'deb [arch={{host_arch}}] https://download.docker.com/linux/ubuntu {{ansible_distribution_release}} stable'
+        repo: 'deb [arch={{host_arch}}] https://download.docker.com/linux/ubuntu {{ansible_distribution_release}} edge'
```

Comment:
stable/edge is what we want to parameterize.  The default should be stable
Also, this will need to be opened against the repo here https://gerrit.linuxfoundation.org/infra/#/admin/projects/ansible/roles/docker-install as this is just a mirror.
Talk again on slack.